Osiris/Hacks/Misc.cpp

```diff
@@ -504,3 +504,27 @@ void Misc::killMessage(GameEvent& event) noexcept
     cmd += "\"";
     interfaces.engine->clientCmdUnrestricted(cmd.c_str());
 }
+
+void Misc::teamDamageCounter(GameEvent* event) noexcept {
+    if (!event || !interfaces.engine->isInGame()) return;
+
+    PlayerInfo pinfo;
+    interfaces.engine->getPlayerInfo(interfaces.engine->getLocalPlayer(), pinfo);
+
+    if (event->getInt("attacker") == pinfo.userId) { // we attacked.
```

Comment:
**interfaces.engine->getPlayerForUserID(event->getInt("attacker")) == interfaces.engine->getLocalPlayer()**
should work as well and does not involve PlayerInfo